utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -59,6 +74,22 @@ inline T validate_pixel_range(T pixel)
     return pixel;
 }
 
+// replicates the last image in a batch to fill the remaining images in a batch
+void replicate_last_image_to_fill_batch(const string& lastFilePath, vector<string>& imageNamesPath, vector<string>& imageNames, const string& lastFileName, int noOfImages, int batchCount)
```

Comment:
Align the params of the function
int batchCount) should be inline with (const string& lastFilePath

Applicable to all such instances in the file

Reply:
Aligment is proper, Not required